src/types/algoliasearch.ts

```diff
@@ -38,7 +38,10 @@ export type SearchResponse<
   THit
 > = DefaultSearchClient extends DummySearchClientV4
   ? SearchResponseV4<THit>
-  : SearchResponseV3<THit> & { appliedRelevancyStrictness?: number };
+  : SearchResponseV3<THit> & {
+      appliedRelevancyStrictness?: number;
+      nbSortedHits?: number;
+    };
```

Comment:
maybe put this interface in its own type called SearchResponsePatchV3 or something?

Reply:
as this is only related to virtual replica indices with the `relevant sort` option, is `SearchWithRelevantSortResponseV3` good?

Reply:
maybe RelevantSortResponse

Reply:
Added in https://github.com/algolia/instantsearch.js/pull/4681/commits/96060a6f2a852eb904ebed98e98400305ba2da58